modules/terraform/apply.go

```diff
@@ -60,3 +62,33 @@ func TgApplyAllE(t testing.TestingT, options *Options) (string, error) {
 
 	return RunTerraformCommandE(t, options, FormatArgs(options, "apply-all", "-input=false", "-lock=false", "-auto-approve")...)
 }
+
+// ApplyAndIdempotent runs terraform init and apply with the given options and return stdout/stderr from the apply command. It then runs
+// plan again and will fail the test if plan requires additional changes. Note that this method does NOT call destroy and assumes
+// the caller is responsible for cleaning up any resources created by running apply.
+func ApplyAndIdempotent(t testing.TestingT, options *Options) string {
+
+	out, err := ApplyAndIdempotentE(t, options)
+	require.NoError(t, err)
+
+	return out
+}
+
+// ApplyAndIdempotentE runs terraform init and apply with the given options and return stdout/stderr from the apply command. It then runs
+// plan again and will fail the test if plan requires additional changes. Note that this method does NOT call destroy and assumes
+// the caller is responsible for cleaning up any resources created by running apply.
+func ApplyAndIdempotentE(t testing.TestingT, options *Options) (string, error) {
+	out, err := InitAndApplyE(t, options)
```

Comment:
Should only call `Apply`. If you want a version that calls `init` and `apply`, it would have to be called something like `InitAndApplyAndIdempotent`.

Reply:
@brikis98 I changed the call to be `Apply` only. Do you think we should offer one that init's as well?

Reply:
I went ahead and created one as that follows the previous code style.